data/helpers.d/network

```diff
@@ -0,0 +1,18 @@
+# Normalize the path syntax
+# Check the slash at the beginning of path and its absence at ending
+#
+# example: path=$(ynh_normalize_path $path)
+#
+# usage: ynh_normalize_path path_to_normalize
+# | arg: path_to_normalize - Path to normalize before using it
+ynh_normalize_path () {
```

Comment:
This fonction should probably be named ynh_normalize_url_path but I'm not even 100% sure which is a problem.